.github/workflows/lint.yaml

```diff
@@ -1,12 +1,8 @@
 on:
   push:
-    branches:
-      - main
-      - master
+    branches: [main, master]
```

Comment:
any reason to keep `master` here? just to keep perfectly synced with the upstream source?

Reply:
Yes, no other reason. Can also remove it. Fine either way.

Reply:
SGTM, thanks for clarifying